src/modules/send/send-link-item.domain.ts

```diff
@@ -71,7 +71,6 @@ export class SendLinkItem implements SendLinkItemAttributes {
       createdAt: this.createdAt,
       updatedAt: this.updatedAt,
       version: this.version,
-      hashedPassword: this.hashedPassword,
```

Comment:
removed the `hashedPassword` from here since there are endpoints that return the serialized model